components/server/ee/src/billing/entitlement-service-ubp.ts

```diff
@@ -118,8 +118,9 @@ export class EntitlementServiceUBP implements EntitlementService {
      * @param user
      */
     async limitNetworkConnections(user: User, date: Date): Promise<boolean> {
-        const hasPaidPlan = await this.hasPaidSubscription(user, date);
-        return !hasPaidPlan;
+        // gpl: Because with the current payment handling (pay-after-use) having a "paid" plan is not a good enough classifier for trushworthyness atm.
+        // We're looking into improving this, but for the meantime we limit network connections for everybody to reduce the impact of abuse.
+        return true;
```

Comment:
@kylos101 All users without any subscription ("free tier") are on UBP now. :heavy_check_mark: 

:left_speech_bubble: You see me twisting your words a bit, because there is one subscription ("Professional Open Source", `free-open-source`) which we hand out to some, so they are still with Chargebee. Those people are [not](https://github.com/gitpod-io/gitpod/blob/0c0178b610b3d46a1703516cd7ac494c187f6055/components/server/ee/src/billing/entitlement-service-chargebee.ts#L199) [limited](https://github.com/gitpod-io/gitpod/blob/612cd936fd2a6c13ad1ae39b01c553a5a2728d38/components/gitpod-protocol/src/plans.ts#L486) atm. :hole: 

But we have to replace that anyway in the upcoming weeks. :lotus_position: